### PR TITLE
Feat/spent so far page

### PR DESCRIPTION
**Pull request by dsidous**
Build page

### CODE REVIEW

File: src/components/organisms/PieChart/index.js [hunk 1]
```diff
@@ -17,27 +17,18 @@ const styles = StyleSheet.create({
```
Comment (davidc94) [inline src/components/organisms/PieChart/index.js]: remove this file

File: src/components/organisms/SpentSoFar/AllSpentList.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import React from 'react';
```
Comment (davidc94) [inline src/components/organisms/SpentSoFar/AllSpentList.js:1]: snapshot test

File: src/components/organisms/SpentSoFar/DayToDaySpentList.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
+import React from 'react';
```
Comment (davidc94) [inline src/components/organisms/SpentSoFar/DayToDaySpentList.js:1]: snapshot test

File: src/components/organisms/SpentSoFar/RegularSpentList.js [hunk 4]
```diff
@@ -0,0 +1,50 @@
+import React from 'react';
```
Comment (davidc94) [inline src/components/organisms/SpentSoFar/RegularSpentList.js:1]: snapshot test

File: src/components/organisms/SpentSoFar/SpentDetails.js [hunk 5]
```diff
@@ -0,0 +1,61 @@
+import React, { Component } from 'react';
```
Comment (davidc94) [inline src/components/organisms/SpentSoFar/SpentDetails.js:1]: add test

File: src/components/pages/Dashboard/index.js [hunk 6]
```diff
@@ -1,14 +1,19 @@
...
 import SpentSoFar from '../../organisms/SpentSoFar';
 import Income from '../../organisms/Income';
+import { FETCH_BANK_BALANCE, FETCH_SPENDING } from '../../../store/constants';
+
+class Dashboard extends PureComponent {
 
-export default class Dashboard extends PureComponent {
   componentDidMount() {
     StatusBar.setBackgroundColor('#000000');
+    this.props.dispatch({ type: FETCH_BANK_BALANCE });
```
Comment (davidc94) [inline src/components/pages/Dashboard/index.js:15]: dispatch income from here

File: src/components/pages/Setup/index.js [hunk 7]
```diff
@@ -32,13 +32,14 @@ export default class Setup extends PureComponent {
...
           </StyledText>
-          <StyledButton 
-            className="button"
-            style={button}
-            onPress={() => this.props.navigation.navigate('ChooseBank')}
-          >
-            Continue
+          <View style={{ paddingHorizontal: 80, marginTop: 30 }}>
+            <StyledButton
+              className="button"
```
Comment (davidc94) [inline src/components/pages/Setup/index.js]: remove className

File: src/store/dashboard/spending/sagas.js [hunk 8]
```diff
@@ -0,0 +1,15 @@
+import { put, takeLatest } from 'redux-saga/effects';
+import { FETCH_SPENDING, FETCH_SPENDING_FULFILLED } from '../../constants';
+import data from '../../../data/spending.json';
+
+export function* fetchingSpending(action, actionData = data) {
```
Comment (davidc94) [inline src/store/dashboard/spending/sagas.js]: maybe change actionData to spendingData?

File: src/components/organisms/PieChart/index.js (hunk 1 above)
Comment (dsidous) [reply src/components/organisms/PieChart/index.js]: Done

File: src/components/organisms/SpentSoFar/AllSpentList.js (hunk 2 above)
Comment (dsidous) [reply src/components/organisms/SpentSoFar/AllSpentList.js:1]: Done

File: src/components/organisms/SpentSoFar/DayToDaySpentList.js (hunk 3 above)
Comment (dsidous) [reply src/components/organisms/SpentSoFar/DayToDaySpentList.js:1]: Done

File: src/components/organisms/SpentSoFar/RegularSpentList.js (hunk 4 above)
Comment (dsidous) [reply src/components/organisms/SpentSoFar/RegularSpentList.js:1]: Done

File: src/components/organisms/SpentSoFar/SpentDetails.js (hunk 5 above)
Comment (dsidous) [reply src/components/organisms/SpentSoFar/SpentDetails.js:1]: Done

File: src/components/pages/Dashboard/index.js (hunk 6 above)
Comment (dsidous) [reply src/components/pages/Dashboard/index.js:15]: Done

File: src/components/pages/Setup/index.js (hunk 7 above)
Comment (dsidous) [reply src/components/pages/Setup/index.js]: Done

File: src/store/dashboard/spending/sagas.js (hunk 8 above)
Comment (dsidous) [reply src/store/dashboard/spending/sagas.js]: Done

**Review by maciejk77 — APPROVED** [review]

**Review by maciejk77 — APPROVED** [review]

**Review by maciejk77 — APPROVED** [review]

**Pull request merged**